### ISSUE TITLE
Há necesidade desse repositório?

### ISSUE THREAD

**Issue by nicholasess** [training-center/guilds#1]
Me veio essa pergunta, pois, a sensação de abraçar o mundo é meio perigoso.
Se o CT quer ajudar as pessoas no desenvolvimento de softwares e englobando o desenvolvimento pessoal, esse grupo não condiz com as diretrizes do mesmo.

**Comment by woliveiras** [comment]
Boa questão, mano!

Quando rolou a conversa no Slack eu me animei e sai criando, mas realmente não condiz com o grupo.

Porém é interessante levar ao restante da comunidade para ver se realmente devemos excluir esse projeto.

Vou mandar lá no #general. :metal:

**Comment by lflimeira** [comment]
Quando a galera estava cogitando essa ideia no Slack eu fiz o mesmo questionamento e surgiram alguns pontos:

- Mais uma porta para pessoas entrarem no TC.
- Uma forma de criar interação entre os integrantes do TC.
- As pessoas poderiam conversar sobre games e até mesmo desenvolvimento de games.

Quanto a se o repositório não condiz com as diretrizes, eu acredito que não muito da forma como está, mas se for pensar dessa forma o CodeHouse tbm n tem muito certo? E nem por isso ele deixa de ser valido e ser um projeto bacana. Talvez se trabalhar alguma ideia para esse projeto podemos deixar com um foco com os objetivos do TC.

**Comment by nicholasess** [comment]
Se for focado em desenvolvimento de games, ele seria um game-study. Além disso, alguém teria que liderar esse repositório, que é a forma que o @woliveiras mencionou num post recente...

**Comment by lflimeira** [comment]
Exato, foi por isso que eu citei fazer algumas modificações, e não simplesmente acabar com o projeto. Vamos esperar o feedback do pessoal e ver as ideias, quem sabe modificar para um grupo de estudos de fato.

Mas como eu citei, ele não é focado em desenvolvimento, ele seria uma forma de levar o TC para outros lugares, foi assim que ele foi idealizado.

**Comment by luxu** [comment]
Pra mim como num ocupa espaço poderia deixá-lo q pode não conduzer com o CT mas também pode "alimentar" a imaginação do pessoal.

**Comment by woliveiras** [comment]
Os pontos do @lflimeira são muito importantes: é legal ter algo pro grupo quebrar aquele fluxo de sempre fazer o mesmo. Isso une mais a galera. 

É igual ter um meetup que sempre acontece em formato de palestra e de repente fazer em formato de mesa redonda. O pessoal se motiva a continuar participando.

Acho que esse projeto é importante pra isso: unir a galera gamer da nossa comunidade para se divertirem juntos fora da seriedade do grupo.

Aqui ficaria como um canal de entrada pra encontrar o Discord, talvez transformar em um GH Pages e não deixar em formato de repositório. O que acham?

**Comment by lflimeira** [comment]
A ideia do GitHub Pages é uma boa, e já aproveitar e deixar com os objetivos do projeto claro. E  com isso já abrir portas para o pessoal se mobilizar para fazer eventos ou incentivar a galera a programar games ou algo do tipo, tem iniciativas bem bacanas nesse quesito.

**Comment by nicholasess** [comment]
Não sei qual é o momento do CT agora, pois, se não tiver um planejamento bem estruturado, terão diversos projetos aleatórios. Não digo que é ruim, acredito que as coisas podem evoluir gradativamente para não frustrar ou criar expectativas demais.

**Comment by woliveiras** [comment]
Galera, como este projeto não está tendo movimentação vamos ter que fechar ele. Não faz sentido, realmente, manter algo que não está tendo movimento e não teremos braços para gerenciar.

Valeu, @nicholasess

**Comment by nicholasess** [comment]
Por nada @woliveiras  hahaha
